server/datastore/mysql/migrations/tables/20220307104655_AddHostDeviceAuth.go

```diff
@@ -16,7 +16,7 @@ func Up_20220307104655(tx *sql.Tx) error {
         host_id int(10) UNSIGNED NOT NULL,
         token VARCHAR(255) NOT NULL,
         PRIMARY KEY (host_id),
-        INDEX idx_host_device_auth_token (token)
+        UNIQUE INDEX idx_host_device_auth_token (token)
```

Comment:
The token must be unique. Let me know if you feel it's going to cause too much trouble to change an existing migration file - it definitely hasn't been released in the wild and would at most require a `make db-reset` for local development environments, but I'm not sure if it's a big issue for other internal envs like load testing.

Reply:
Should be fine, it's certainly unreleased.